rules/60-sensor.rules

```diff
@@ -7,4 +7,8 @@ SUBSYSTEM=="iio", KERNEL=="iio*", SUBSYSTEMS=="usb|i2c", \
   IMPORT{builtin}="hwdb 'sensor:modalias:$attr{modalias}:$attr{[dmi/id]modalias}'", \
   GOTO="sensor_end"
 
+SUBSYSTEM=="input", ENV{ID_INPUT_ACCELEROMETER}=="1", SUBSYSTEMS=="acpi", \
+  IMPORT{builtin}="hwdb 'sensor:modalias:acpi:$attr{hid}::$attr{[dmi/id]modalias}'", \
```

Comment:
well, but this is actually unnecessary...

modalias strings generally start and end in a colon, so that you can match against parts of it easily with wildcards. i.e. if you want to match on the existance of a field "foo" in it, you can just write "*:foo:*", instead of having to specify three matches: "foo:*", "*:foo" and "*:foo:*"...

Now, the iio rule prefixes and suffixes the `$attr{modalias}` part with colons to prepare for the case where the attribute might not be set, to ensure that the fields before and after it have at least one colon even in that case. The fact that in the usual case this then results in two colons is just a side effect that is OK, but not really the primary goal here...

hence, i don't think we should duplicate the double colons here, there's little benefit in it...